script/delete-modules-f.sh

```diff
@@ -0,0 +1,24 @@
+# get  argument or default to "."
```

Comment:
I'm not immediately sure what the `-f` means in this filename. Consider removing that, or expanding it to be more descriptive (`-force`?)

Reply:
The `f` was meant to signify `folder`, but update this file and script name accordingly